src/panel/NoResultMessage.jsx

```diff
@@ -0,0 +1,20 @@
+/* globals _ */
+import React from 'react';
+
+const NoResultMessage = () => {
+  return (
+    <>
+      <p className="u-center u-mb-xs u-text--smallTitle">
+        {_('Sorry, we could not find this place 🏝️', 'suggest')}
+      </p>
+      <p className="u-center u-text--subtitle u-mb-l">
+        {_(
+          'Please try to correct your query or rewrite it with more details about the location (city, country, ...)',
```

Comment:
Sorry for being such a typo-nerd, but let's use the real ellipsis character `…` here instead of three dots.